internal/findprocess/findprocess_unix.go

```diff
@@ -7,8 +7,8 @@ import (
 	"syscall"
 )
 
-func findProcess(pid int) (process *os.Process, err error) {
-	process, err = os.FindProcess(pid)
+func findProcess(pid int) (*os.Process, error) {
+	process, err := os.FindProcess(pid)
```

Comment:
Not directly related to this PR, but err handling can be dropped for unix, i.e.
```go
// os.FindProcess never returns an error on unix
process, _ := os.FindProcess(pid)
```
should suffice